content/stats/measure.md

```diff
@@ -139,7 +139,31 @@ int main(int argc, char** argv) {
             {{LinesMeasure(), 238}},
             {{BytesInMeasure(), 7000}});
 }
-{{</highlight>}}
+```
+
+```js
+import { Stats, MeasureUnit } from "@opencensus/core";
+
+// Our Stats manager
+const stats = new Stats();
+
+const mLatencyMs = stats.createMeasureDouble("latency", MeasureUnit.MS, "The latency in milliseconds");
+const mLinesIn = stats.createMeasureInt64("size", MeasureUnit.UNIT, "The number of lines processed");
```

Comment:
This measure is not needed because in other samples we are merely applying a count aggregation to mLatencyMs which is recorded on every line. Please remove `mLinesIn`